### ISSUE TITLE
How can i change a Distribution?

### ISSUE THREAD

**Issue by CAI23sbP** [DLR-RM/stable-baselines3#2032]
### ❓ Question

How are you @araffin ?
I have two questions
1. If I want to change distribution , how can I do it?

Here is my situation 
I made an agent that was based on PPO(diagonal dist) by imitation learning library( behavioral cloning)
And then, Finetuning the pretrained model use  a 'gSDE' (StatedependentDist)
It is right to change a sde_sample_freq from 1 to n?

2. How can i create a normalized observation(for demonstration data)? Is it okay to use a VecNormEnv?
Might i get any hit about the above issues?

Thank you!

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] If code there is, it is [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014)
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by araffin** [comment]
>  It is right to change a sde_sample_freq from 1 to n?

should be fine

> Is it okay to use a VecNormEnv?

that's one solution. But if you have a fixed dataset, you can precompute the normalization.

**Comment by CAI23sbP** [comment]
Thank you!
